keychain/src/mnemonic.rs

```diff
@@ -124,7 +124,7 @@ pub fn from_entropy(entropy: &Vec<u8>) -> Result<String, Error> {
 
 	let mut hash = [0; 32];
 	let mut sha2sum = Sha256::default();
-	sha2sum.input(&entropy.clone());
+	sha2sum.input(&(*entropy));
```

Comment:
This `&(*foo)` looks weird - what's this actually doing?
I'm guessing `entropy` cannot just be used directly, but not immediately clear to me why not.

Reply:
Indeed, fixed.